src/components/WebexSettings/Tabs.jsx

```diff
@@ -18,22 +19,52 @@ import webexComponentClasses from '../helpers';
  * @param {string} props.selected  Specifies which tab is active
  * @param {object} props.style  Custom style to apply
  * @param {Tab[]} props.tabs  The tabs to be rendered
+ * @param {number} props.tabIndex  Tab index
  * @returns {object} JSX of the component
- *
  */
 export default function Tabs({
   className,
   onSelect,
   selected,
   style,
   tabs,
+  tabIndex,
 }) {
   const {content} = tabs.find((tab) => tab.key === selected);
   const cssClasses = webexComponentClasses('tabs', className);
+  const tabListRef = useRef();
+  const leftPress = useKeyPress('ArrowLeft', tabListRef);
+  const rightPress = useKeyPress('ArrowRight', tabListRef);
+
+  useEffect(() => {
+    const selectedTabIndex = (tabs || []).findIndex((tab) => tab.key === selected);
+
+    if (leftPress) {
+      if (selectedTabIndex > 0) {
+        onSelect(tabs[selectedTabIndex - 1].key);
+      } else {
+        onSelect(tabs[tabs.length - 1].key);
+      }
+    }
+    // eslint-disable-next-line
+  }, [leftPress]);
+
+  useEffect(() => {
+    const selectedTabIndex = (tabs || []).findIndex((tab) => tab.key === selected);
+
+    if (rightPress) {
+      if (selectedTabIndex < tabs.length - 1) {
+        onSelect(tabs[selectedTabIndex + 1].key);
+      } else {
+        onSelect(tabs[0].key);
+      }
+    }
+    // eslint-disable-next-line
+  }, [rightPress]);
 
   return (
     <div className={cssClasses} style={style}>
-      <ul className="tabs-list">
+      <ul className="tabs-list" tabIndex={tabIndex} ref={tabListRef}>
```

Comment:
I'm not sure why the `ul` needs a tabIndex

Reply:
We need the tabIndex for the user to be able to focus the whole list of tabs by pressing tab so he can move trough them with arrow keys. This will also be useful for hints as we will need to let the user know he jumped on the tabs section.